pkg/libgit2

```diff
@@ -0,0 +1,41 @@
+[mirrors]
+https://github.com/libgit2/libgit2/archive/v0.27.4.tar.gz
+
+[vars]
+filesize=4772254
+sha512=d27db86eb1b9f0d4057f8538ba1985ee76c3ca106e57d417fa9bff79d575f91a07ad28693112b58dc1d61d68116a82e6a145f12276158f2806b6c4964d741f61
+tardir=libgit2-0.27.4
```

Comment:
could you add a tarball directive here, to prevent the tarball from being saved as v0.27.tar.gz

see pkg/john as example

Reply:
ok